template/roc.config.js

```diff
@@ -10,5 +10,25 @@ module.exports = {
       reducers: 'src/redux/reducers.js',
       routes: 'src/routes/index.js',
     },
+    test: {
+      tests: {
+        pattern: /test/i,
```

Comment:
I think that this is the same as the the default from a functional perspective, so this should not be needed.